src/unix/ftrace.c

```diff
@@ -737,6 +762,7 @@ function_graph_toggle(boolean enable)
 static void
 print_switch_event(struct ftrace_printer * p, struct rbuf_entry_switch * sw)
 {
+    return;
```

Comment:
debug cruft?

Reply:
Sort of--thread switches aren't correct yet, so it screws up analysis tools with incomplete or misplaced messages. This is to be fixed later with the thread switches. I just changed it to be ifdef'd out and added a comment to explain what's going on so it's not so hacky.